sqlalchemy-stubs/sql/sqltypes.pyi

```diff
@@ -77,47 +85,52 @@ class Float(Numeric):
     decimal_return_scale: Any = ...
     def __init__(self, precision: Optional[Any] = ..., asdecimal: bool = ...,
                  decimal_return_scale: Optional[Any] = ..., **kwargs) -> None: ...
-    def result_processor(self, dialect, coltype): ...
+    def result_processor(self, dialect: Dialect, coltype: Any): ...
 
 class DateTime(_DateAffinity, TypeEngine[datetime]):
     __visit_name__: str = ...
     timezone: Any = ...
     def __init__(self, timezone: bool = ...) -> None: ...
     def get_dbapi_type(self, dbapi): ...
     @property
-    def python_type(self): ...
+    def python_type(self) -> Type[datetime]: ...
 
 class Date(_DateAffinity, TypeEngine[date]):
     __visit_name__: str = ...
     def get_dbapi_type(self, dbapi): ...
     @property
-    def python_type(self): ...
+    def python_type(self) -> Type[date]: ...
 
 class Time(_DateAffinity, TypeEngine[time]):
     __visit_name__: str = ...
-    timezone: Any = ...
+    timezone: bool = ...
     def __init__(self, timezone: bool = ...) -> None: ...
     def get_dbapi_type(self, dbapi): ...
     @property
-    def python_type(self): ...
+    def python_type(self) -> Type[time]: ...
 
-class _Binary(TypeEngine[bytes]):
-    length: Any = ...
-    def __init__(self, length: Optional[Any] = ...) -> None: ...
-    def literal_processor(self, dialect): ...
+if sys.version_info[0] < 3:
+    _BinaryType = str
+else:
+    _BinaryType = bytes
```

Comment:
Is this really needed? I though `bytes` is always the binary type, in Python 2 it is just an alias for `str`

Reply:
I was just following what sqlalchemy does in `sqlalchemy.util.compat`. If you think this should just be bytes, I can change it to that.

Reply:
OK, let us keep it then.